examples/translation-job-items.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
I don't really get the purpose of this file.

Should we rename it to "translation-item-types.json".
But it's not really an endpoint we need, right? Or what is the use case?